### PR TITLE
[velero] Improve docs and add examples 

### PR DESCRIPTION
**Pull request by MaxymVlasov**
#### Special notes for your reviewer:
I also can add Terraform 0.13 code for setup AWS and vSphere from zero, but not sure that it should be here.

P.S. Sad, but helm-chart not be used for up Velero 1.5.1 yet, so examples based on 1.4.2.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[velero]`)

### CODE REVIEW

**Comment by jenting** [comment]
I'm sure not is it good to put an example in the released helm chart :thinking:  _or_ would it be good to add the examples to the README.
What are other reviewers think? @nrb @carlisia @ashish-amarnath @cpanato

File: charts/velero/README.md [hunk 1]
```diff
@@ -155,3 +180,7 @@ helm delete <RELEASE NAME> --purge
 ```bash
 helm delete <RELEASE NAME> -n <YOUR NAMESPACE>
 ```
+
+## Chart configuration examples
+
+In [examples folder](./examples/) located some basic scenario for different plugins. It does not include settings that you need make in your cloud provider, see plugin README for details.
```
Comment (cpanato) [inline charts/velero/README.md:186]: I like this, thanks for adding this! :)

I would add a link here to go straight to the AWS example and to the vsphere as well

Comment (MaxymVlasov) [reply charts/velero/README.md:191]: Done

**Comment by nrb** [comment]
> Looks like the AWS plugin not support Velero v1.5 yet, so examples stay for v1.4.

It does, I'll update the compatibility chart.

**Comment by MaxymVlasov** [comment]
Okay, maybe it was another issue, because I try up and run 1.5.2 yesterday, after full removal of 1.4.2. but `velero version` not found Velero Server version at all, when Velero pod works fine and shedule triggered backup.

All that I changed - chart version to latest and tag to 1.5.2. All AWS infra was recreated from scratch.

**Comment by carlisia** [comment]
I love the intention of this PR and some things are useful, but replicating the documentation in this repo creates a maintenance burden because we will need to keep it in sync with the original documentation, and I'm not sure it buys us much. I'll take a closer look at this today.

**Comment by mkilchhofer** [comment]
> I'm sure not is it good to put an example in the released helm chart 🤔 _or_ would it be good to add the examples to the README.

When the examples directory is ignored from packaging by adding it to the `.helmignore` file, this would be okay.

**Comment by MaxymVlasov** [comment]
Added `examples/` to `.helmignore`

File: charts/velero/examples/vsphere/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+apiVersion: v1
+kind: Secret
+metadata:
+  name: velero-aws-creds
+data:
+  # echo -n "[default]
+  # aws_access_key_id=ACCESSKEY
+  # aws_secret_access_key=SECRETACCESSKEY
+  # " | base64
+  cloud: W2RlZmF1bHRdCmF3c19hY2Nlc3Nfa2V5X2lkPUFDQ0VTU0tFWQphd3Nfc2VjcmV0X2FjY2Vzc19rZXk9U0VDUkVUQUNDRVNTS0VZCg==
```
Comment (mkilchhofer) [inline charts/velero/examples/vsphere/secret.yaml:11]: I would use stringData in this example here, then you do not need this comment to show whats encoded in this base64 string:
```yaml
stringData:
  cloud: |
    [default]
    aws_access_key_id=ACCESSKEY
    aws_secret_access_key=SECRETACCESSKEY
```

File: charts/velero/examples/aws/secret.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+apiVersion: v1
+kind: Secret
+metadata:
+  name: velero-aws-creds
+data:
+  # echo -n "[default]
+  # aws_access_key_id=ACCESSKEY
+  # aws_secret_access_key=SECRETACCESSKEY
+  # " | base64
+  cloud: W2RlZmF1bHRdCmF3c19hY2Nlc3Nfa2V5X2lkPUFDQ0VTU0tFWQphd3Nfc2VjcmV0X2FjY2Vzc19rZXk9U0VDUkVUQUNDRVNTS0VZCg==
```
Comment (mkilchhofer) [inline charts/velero/examples/aws/secret.yaml:11]: See other comment regarding the usage of `stringData`

**Comment by jenting** [comment]
> I love the intention of this PR and some things are useful, but replicating the documentation in this repo creates a maintenance burden because we will need to keep it in sync with the original documentation, and I'm not sure it buys us much. I'll take a closer look at this today.

@carlisia,
I'd prefer we could add these examples to https://github.com/vmware-tanzu/helm-charts/tree/main/charts/velero/ci
So, we could make sure these examples work well by CI testing.

File: charts/velero/examples/aws/secret.yaml (hunk 3 above)
Comment (MaxymVlasov) [reply charts/velero/examples/aws/secret.yaml]: Thanks, done

File: charts/velero/examples/vsphere/secret.yaml (hunk 2 above)
Comment (MaxymVlasov) [reply charts/velero/examples/vsphere/secret.yaml]: Thanks, done

**Comment by MaxymVlasov** [comment]
Well, I think now it's completely outdated so there is no sense to keep that PR open

**Pull request closed without merging**